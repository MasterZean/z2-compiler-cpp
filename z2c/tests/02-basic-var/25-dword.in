val a = 0u32
val b = 65536u32
val c = 4294967294u32
val d = 4294967295u32