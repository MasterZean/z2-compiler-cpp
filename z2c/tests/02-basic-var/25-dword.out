uint32 a = 0u;
uint32 b = 65536u;
uint32 c = 4294967294u;
uint32 d = 4294967295u;
